doc/dune

```diff
@@ -6,7 +6,7 @@
   (glob_files *.ml*)
   (package odoc))
  (enabled_if
-  (> %{ocaml_version} 4.11))
+  (> %{ocaml_version} 4.11.2))
```

Comment:
What's the purpose of this?

Reply:
This is effectively disabling the test in CI.
Removing this commit and rebasing the PR on the current master solves the problem: https://github.com/Julow/odoc/actions/runs/906918515

Reply:
great, thanks!